src_assets/common/assets/web/public/assets/locale/en.json

```diff
@@ -158,6 +158,8 @@
     "dd_config_label": "Device configuration",
     "dd_config_revert_delay": "Config revert delay",
     "dd_config_revert_delay_desc": "Additional delay in milliseconds to wait before reverting configuration when the app has been closed or the last session terminated. Main purpose is to provide a smoother transition when quickly switching between apps.",
+    "dd_config_revert_on_disconnect": "Config revert on disconnect",
+    "dd_config_revert_on_disconnect_desc": "Revert configuration upon disconnect of all clients instead of session termination.",
```

Comment:
```suggestion
    "dd_config_revert_on_disconnect_desc": "Revert configuration upon disconnect of all clients instead of app close or last session termination.",
```